RamilyaNigmatullina/pdp-3
# Add search by tags

# Summary

Add ability to search posts by tags

https://github.com/RamilyaNigmatullina/pdp-3/projects/1#card-19568348

# Test plan

List of steps to manually test introduced functionality:

* Go to Application
* Create and approve post with tag lorem
* Create and approve post with tag ipsum
* Go to the main page
* Fill in search field with lorem word
* Click on Find button
* Check there is only one post with tag lorem

# Review notes

While reviewing pull-request (especially when it's your pull-request),
please make sure that:

- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- DB schema changes reflect new migrations
- newly introduces DB fields have indexes and constraints
- routes are RESTful, no useless routes
- there are no missed files (migrations, view templates)
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes

# Deploy notes

None.
